<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotDialog</class>
 <widget class="QDialog" name="BotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="l_username">
     <property name="text">
      <string>UserName</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="le_username"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l_password">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="le_password"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="l_proxy">
     <property name="text">
      <string>Proxy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="le_proxy"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pb_ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
